libs/utils/include/utils/compiler.h

```diff
@@ -192,4 +200,10 @@ typedef SSIZE_T ssize_t;
     #define IMPORTSYMB
 #endif
 
+#if _MSC_VER && !defined(__PRETTY_FUNCTION__)
```

Comment:
this should be `#if defined(_MSC_VER) &&` ... to avoid warnings on non-msvc compilers

Reply:
done